conda-forge.yml

```diff
@@ -1,3 +1,3 @@
 # linux_ppc64le is explicitly removed again, see https://github.com/conda-forge/arrow-cpp-feedstock/issues/143
-provider: {linux_aarch64: default, win: azure}
+provider: {linux_aarch64: default, linux_ppc64le: azure, win: azure}
```

Comment:
So using `azure` means it is cross-compiled. `default` is built natively on Travis CI. Would suggest trying `default`, but we can go with cross-compiled if that doesn't work.